technical-docs/designs/introspection/deploymentDashboardCaching.md

```diff
@@ -101,6 +119,9 @@ is in hundreds and not in thousands. Otherwise, we have to increase the _RAM_
 
 ## 6. Documentation
 
-How to change the refresh duration
+How to change the refresh duration: Check the deployment environment variables
+passed in through the docker image or helm chart, and add a variable
+`REACT_APP_CACHE_REFRESH_IN_SEC` set it to let's say `60` for a 60 seconds
```

Comment:
Once this is added, this should also set the default `refreshRate`  property in the frontend dashboard component state

Reply:
@evanlouie When the dashboard launches, should we apply this env setting by default to the `refreshRate` property in the component state?

Reply:
there are two refresh rate settings. One for backend and one for frontend. `REACT_APP_CACHE_REFRESH_IN_SEC` is for backend.